tpmutil/tbs/tbs_windows_test.go

```diff
@@ -111,3 +111,78 @@ func TestSubmitCommandLongResponse(t *testing.T) {
 		t.Fatalf("Got response of %v, expected %v", rawResponse, expectedGetRandomRawResponse)
 	}
 }
+
+// Get Storage owner authorization delegation blob
+func TestGetStorageOwnerAuth(t *testing.T) {
+	ctx := getContext(t)
+	defer ctx.Close()
+
+	authBufferLength, err := ctx.GetOwnerAuth(Storage20Authorization, nil)
+	if err == ErrOwnerauthNotFound || err == ErrInternalError {
+		expectedErrorHint := "An internal error occurred.  Are you running with Administrative permissions?"
+		if err == ErrOwnerauthNotFound {
+			expectedErrorHint = "Delegation blob not available in the registry."
+		}
+		t.Logf("Skipping retrieval of Storage authorization; %v", expectedErrorHint)
+		t.SkipNow()
+	} else if err != nil {
+		t.Fatalf("Failed to get Storage authorization delegation blob size: %v", err)
+	}
+
+	storageOwnerAuth := make([]byte, authBufferLength)
+	if _, err := ctx.GetOwnerAuth(Storage20Authorization, storageOwnerAuth); err != nil && err != ErrOwnerauthNotFound {
+		t.Fatalf("Failed to retrieve Storage Authorization delegation blob from the registry: %v", err)
+	} else if err == ErrOwnerauthNotFound {
+		t.Log("Skipping retrieval of Storage authorization; Delegation blob not available in the registry.")
+		t.SkipNow()
+	}
+}
```

Comment:
Might make sense to try using the auth value to make sure it came through correctly (i.e., doesn't need base64 decoded or something)